src/api/component.js

```diff
@@ -21,6 +21,7 @@ import getOwnPropertyDescriptors from '../util/get-own-property-descriptors';
 import getPropsMap from '../util/get-props-map';
 import getSetProps from './props';
 import initProps from '../lifecycle/props-init';
+import {isFunction} from '../util/isType';
```

Comment:
nit: semistandard probably didn't catch this as it's consistent with itself, but generally there's spaces around object literals, I think.